cobc/ppparse.y

```diff
@@ -1725,6 +1726,23 @@ lead_trail:
   }
 ;
 
+unquoted_literal:
```

Comment:
I suggest that you directly commit the "unquoted literal" part, along with unquote_alnum and correct whitespace trimming with its Changelog entry to svn, leaving the _actual_ change which this PR is about in a more clean state.

Reply:
I am currently splitting this PR into two commits for the purpose of separating code factorization from the support for literal operands.
About the latter, as of now I failed to locate relevant documentations for Realia, BS2000 and RM-COBOL. For the others I could only trust https://www.microfocus.com/documentation/visual-cobol/vc60/VS2019/HRLHLHCOMP02.html .

Reply:
> About the latter, as of now I failed to locate relevant documentations for Realia, BS2000 and RM-COBOL. For the others I could only trust https://www.microfocus.com/documentation/visual-cobol/vc60/VS2019/HRLHLHCOMP02.html .

That's fine, thank you for taking it that far.

> I am currently splitting this PR into two commits for the purpose of separating code factorization from the support for literal operands.

Thank you, as noted: that first commit could go to svn directly when you consider it clear (or I could have a look at both commits here and you check it in "upstream" with two commits when this PR is review-approved).

Reply:
Ok I've just pushed the two commits here.